packages/db/tsconfig.json

```diff
@@ -2,7 +2,8 @@
   "extends": "../../tsconfig.json",
   "compilerOptions": {
     "outDir": "dist",
-    "declaration": true
+    "declaration": true,
+    "exactOptionalPropertyTypes": true
```

Comment:
I'll create a task on the board regarding telling users in the docs that this should be added.